keyboards/rgbkb/zen/rev1/keymaps/samae/keymap.c

```diff
@@ -0,0 +1,64 @@
+#include QMK_KEYBOARD_H
```

Comment:
Could you add a license header here, please?

Here's a sample one to modify:

https://github.com/qmk/qmk_firmware/blob/940280c43594833e2208d8514452b8ab88c14a9c/quantum/template/base/keymaps/default/keymap.c#L1-L15

Reply:
Sure thing (been a busy time these last days, sorry for the delay)